### ISSUE TITLE
Unable to launch the debugger

### ISSUE THREAD

**Issue by DonJayamanne** [Microsoft/ptvsd#292]
Getting the following error in master when debugging in VSC:
```
Could not connect to None: 60857
Traceback (most recent call last):
  File "/Users/donjayamanne/Desktop/Development/vscode/ptvsd/ptvsd/pydevd/pydevd.py", line 1620, in main
    debugger.connect(host, port)
  File "/Users/donjayamanne/Desktop/Development/vscode/ptvsd/ptvsd/pydevd/pydevd.py", line 326, in connect
    s = start_server(port)
  File "/Users/donjayamanne/Desktop/Development/vscode/ptvsd/ptvsd/wrapper.py", line 1766, in start_server
    server = _create_server(port)
  File "/Users/donjayamanne/Desktop/Development/vscode/ptvsd/ptvsd/wrapper.py", line 1701, in _create_server
    server.bind(('127.0.0.1', port))
OSError: [Errno 48] Address already in u
```

**Comment by DonJayamanne** [comment]
This would indicate the live tests we have today are not sufficient or not mimicking VSC.
@karthiknadig Please could you test the latest master branch as well.

**Comment by DonJayamanne** [comment]
@ericsnowcurrently 

The problem is the `--client` arg is not appended into the list when it should be.

```python
def _run_argv(address, filename, *extra):
    """Convert the given values to an argv that pydevd.main() supports."""
    if '--' in extra:
        pydevd = list(extra[:extra.index('--')])
        extra = list(extra[len(pydevd) + 1:])
    else:
        pydevd = []
        extra = list(extra)

    host, port=address
    print(host)
    #if host is None:
    #    host = '127.0.0.1'
    argv = [
        sys.argv[0],
        '--port', str(port),
    ]
    if host is not None:
        argv.extend([
            '--client', host,
        ])
```

Today I'm launching PTVSD by invoking the function `debug` in `debugger.py`:
```typescript
vspd.debug(filename, port_num, debug_id, debug_options, run_as)
```

**Comment by DonJayamanne** [comment]
@ericsnowcurrently 
When possible, please do look at this issue. Should be simple. I'd fix it, but I'm not sure what the right approach is, as you have commented out the code that initializes the host to `127.0.0.1`.

Oh yes, this PR is quite crucial for VSC, as it breaks all of our PRs (we're pulling the master branch of PTVSD in VSC CI tests , i.e. none of our CI tests pass as a result of this 😄)

**Comment by ericsnowcurrently** [comment]
FWIW, I expect that the culprit is PR #266.

**Comment by DonJayamanne** [comment]
@ericsnowcurrently Assigned two other issues which I believe may be related.

**Comment by ericsnowcurrently** [comment]
@DonJayamanne Please verify that this is (still) fixed now that #306 has been merged.

**Comment by DonJayamanne** [comment]
Fixed
